addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -308,12 +308,28 @@ var Wysiwyg = Widget.extend({
             await this.editor.execCommand(onSaveLinkDialog);
         });
     },
-    openMediaDialog() {
-        var mediaDialog = new weWidgets.MediaDialog(this,
-            {},
-        );
+    openMediaDialog(params) {
+        let mediaDialogOptions = {}
+        let htmlClass = ""
+
+        // Customize the look and behavior of the media modal when
+        // the origin of the media modal is a FA icon double click
+        if (params.origin === "fontAwesomeDoubleClick") {
```

Comment:
Comment ils faisaient avant dans Odoo ?

Reply:
Je sais pas, je vais investiguer. Mais a mon avis il ouvrait la media modal dans 2 code distinct, ce qui n'est pas le cas avec JW.

Je check.

Reply:
Something like:
```js
const VideopickerUserValueWidget = MediapickerUserValueWidget.extend({
    //--------------------------------------------------------------------------
    // Handlers
    //--------------------------------------------------------------------------

    /**
     * @override
     */
    _onEditMedia(ev) {
        // Need a dummy element for the media dialog to modify.
        const dummyEl = document.createElement('iframe');
        const dialog = this._openDialog(dummyEl, {videos: true});
        dialog.on('save', this, data => {
            this._value = data.bgVideoSrc;
            this._onUserValueChange();
        });
    },
});
```
and 
```js
    //--------------------------------------------------------------------------
    // Private
    //--------------------------------------------------------------------------

    /**
     * Creates and opens a media dialog to edit a given element's media.
     *
     * @private
     * @param {HTMLElement} el the element whose media should be edited
     * @param {boolean} [images] whether images should be available
     *   default: false
     * @param {boolean} [videos] whether videos should be available
     *   default: false
     */
    _openDialog(el, {images = false, videos = false}) {
        el.src = this._value;
        const $editable = this.$target.closest('.o_editable');
        const mediaDialog = new weWidgets.MediaDialog(this, {
            noImages: !images,
            noVideos: !videos,
            noIcons: true,
            noDocuments: true,
            isForBgVideo: true,
            'res_model': $editable.data('oe-model'),
            'res_id': $editable.data('oe-id'),
        }, el).open();
        return mediaDialog;
    },
```

Reply:
I would suggest 
```js
mediaDialogOptions = params
```